lib/functions.js

```diff
@@ -204,7 +204,7 @@ function executeHighlightRow(currentPrLine, applicableTest) {
     });
 
     // Prepare comment on click.
-    data.addEventListener('dblclick', function () {
+    data.querySelector('.cc-add-comment').addEventListener('click', function () {
```

Comment:
See if this works with reference links present in tooltip text.

Reply:
@gupta-ashutosh5 - Can you please check this up?